packages/suite/src/views/onboarding/steps/ResetDevice/index.tsx

```diff
@@ -22,103 +27,78 @@ const ResetDeviceStep = () => {
+                            onClick={async () => {
+                                if (isShamirBackupAvailable) {
+                                    await onResetDevice({ backup_type: 0 });
+                                } else {
+                                    await onResetDevice();
+                                }
                             }}
-                            title={<Translation id="SHAMIR_SEED" />}
-                            text={<Translation id="SHAMIR_SEED_DESCRIPTION" />}
-                            button={
-                                <Translation
-                                    id="TR_SELECT_SEED_TYPE"
-                                    values={{ seedType: <Translation id="SHAMIR_SEED" /> }}
-                                />
-                            }
-                            imgSrc="images/svg/seed-card-shamir.svg"
+                            heading={<Translation id="SINGLE_SEED" />}
+                            description={<Translation id="SINGLE_SEED_DESCRIPTION" />}
                         />
-                    </Wrapper.Options>
-                )}
-                {!isShamirBackupAvailable() && (
-                    <Wrapper.Controls>
-                        <OnboardingButton.Cta
-                            data-test="@onboarding/only-backup-option-button"
-                            onClick={() =>
-                                callActionAndGoToNextStep(resetDevice, STEP.ID_SECURITY_STEP)
-                            }
-                        >
-                            <Translation id="TR_CREATE_WALLET" />
-                        </OnboardingButton.Cta>
-                    </Wrapper.Controls>
-                )}
-            </Wrapper.StepBody>
-            <Wrapper.StepFooter>
-                <OnboardingButton.Back onClick={() => goToPreviousStep()}>
-                    <Translation id="TR_BACK" />
-                </OnboardingButton.Back>
-            </Wrapper.StepFooter>
-        </Wrapper.Step>
+                    </OptionWrapper>
+                    {isShamirBackupAvailable && (
+                        <>
+                            <OptionsDivider />
+                            <OptionWrapper>
+                                <NeueOption
+                                    icon="SEED_SHAMIR"
+                                    data-test="@onboarding/shamir-backup-option-button"
+                                    onClick={async () => {
+                                        await onResetDevice({ backup_type: 1 });
+                                    }}
+                                    heading={<Translation id="SHAMIR_SEED" />}
+                                    description={<Translation id="SHAMIR_SEED_DESCRIPTION" />}
+                                />
+                            </OptionWrapper>
+                        </>
+                    )}
+                </OptionsWrapper>
+            ) : undefined}
```

Comment:
Unnecessary else case

Reply:
this else case is very much essential and I'll leave finding out why as an exercise 😁  (will later add comment to it explaining why)